packages/react-server-dom-webpack/src/ReactFlightWebpackNodeRegister.js

```diff
@@ -206,8 +187,8 @@ module.exports = function register() {
           {
             name: {value: name},
```

Comment:
I think this can be removed?

Reply:
I had the same thought and created #26322.

Reply:
I left it intentionally for debugging purposes since these are functions and they're printed with their `name` property. The others either have intentionally empty names or hard coded names like "then".